xblock/test/test_core.py

```diff
@@ -3,16 +3,18 @@
 Tests the fundamentals of XBlocks including - but not limited to -
 metaclassing, field access, caching, serialization, and bulk saves.
 """
+from __future__ import unicode_literals
+from builtins import next, str  # pylint: disable=redefined-builtin
```

Comment:
Huh.  That's what the docs say it's good for, but that hypothetical code that only defines `__next__` (and not `next`) will fail as soon as python 2 tries to use it in a loop.  Seems minimally useful, but no big deal.